editor/source/graphics/GridGraphic.js

```diff
@@ -117,7 +130,8 @@ export default class GridGraphic extends Graphic {
         })
       }
     })
-    this._renderMetrics()
+    this.originalTilesLength = this._tiles.length || 0 // save tiles length so the stats does not have a moving target
+    this.metrics.renderMetrics(this._tiles, this.originalTilesLength)
```

Comment:
Since this same call (arguments and all) is made several times, could we wrap it in a one-liner helper method? It appears to be a general-purpose "update" call, is that right?